crates/task-impls/src/da.rs

```diff
@@ -209,9 +209,11 @@ impl<TYPES: NodeType, I: NodeImplementation<TYPES>, A: ConsensusApi<TYPES, I> +
                 }
 
                 // Record the payload we have promised to make available.
-                consensus
-                    .saved_payloads
-                    .insert(view, Arc::clone(&proposal.data.encoded_transactions));
+                if let Err(e) = consensus
+                    .update_saved_payloads(view, Arc::clone(&proposal.data.encoded_transactions))
+                {
+                    tracing::trace!("{e:?}");
```

Comment:
Wouldn't that be just "()"?

Reply:
Good catch! Incorrect pasting.

Reply:
Actually I should fix where this is copied from as well.